content/en/ambassadors.md

```diff
@@ -17,7 +16,22 @@ The TODO OSPO Ambassador Program encompasses a group of community leaders who pr
 * Provide valuable input and feedback to TODO about community programs and initiatives
 * Advocate OSPO best practices and TODO initiatives globally
 
-## Requirements
+## Meet our Ambassadors!
+
+| Name | Organization | Location | Local Efforts
+| --- | --- | --- | --- |
+| [Angel Ramirez](https://github.com/ar4mirez) | Cuemby | LATAM (Mexico 🇲🇽, Colombia 🇨🇴) | [OSPO Local Meetup LATAM](https://community.linuxfoundation.org/ospo-local-meetup-latam-spanish-speaking/) |
+| [Gergely Csatári](https://github.com/CsatariGergely) | Nokia | Finland 🇫🇮 | [OSPO Local Meetup Helsinki](https://community.linuxfoundation.org/ospo-local-meetup-helsinki/) |
+| [Hitomi Mizugaki](https://github.com/hit0mim)| Cuemby | United States (Florida 🇺🇸) | OSPO Local Meetup Florida |
+| Jose Rodrigez Roa | Transformente | LATAM (Mexico 🇲🇽, Colombia 🇨🇴, Ecuador 🇪🇨, Chile 🇨🇱) | [OSPO Local Meetup LATAM](https://community.linuxfoundation.org/ospo-local-meetup-latam-spanish-speaking/) |
```

Comment:
```suggestion
| [Jose Rodrigez Roa](https://www.linkedin.com/in/joserodriguezroa/) | Transformente | LATAM (Mexico 🇲🇽, Colombia 🇨🇴, Ecuador 🇪🇨, Chile 🇨🇱) | [OSPO Local Meetup LATAM](https://community.linuxfoundation.org/ospo-local-meetup-latam-spanish-speaking/) |
```

Reply:
Updated my commit to include Jose's linkedin.